### PR TITLE
Add more partner subscriptions to live test cleanup pipeline

### PR DESCRIPTION
**Pull request by benbp**
This adds two public cloud subscriptions managed by the ACS and Cosmos partner teams to our live test cleanup job.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

**Comment by benbp** [comment]
Passing pipeline with the new az powershell and subscription changes: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=877495&view=results

File: eng/pipelines/live-test-cleanup.yml [hunk 1]
```diff
@@ -34,6 +39,7 @@ stages:
         displayName: Clean up subscription resources (AzureCloud-Preview)
 
       - pwsh: |
+          eng/common/TestResources/Import-AzModules.ps1
```
Comment (weshaggard) [inline eng/pipelines/live-test-cleanup.yml:42]: Why not import this in the clean-up script?

**Pull request merged**